### PR TITLE
WASAPI audio driver compile fix on mingw

### LINKED ISSUE #18046: Compiling for Mingw fails (Audio Driver)
**Godot version:**
master

**OS/device including version:**
Antergos OS (Arch Linux)

**Issue description:**
Compiling for Mingw fails after recent pull (maybe https://github.com/godotengine/godot/commit/1570a72eee4bc0efb46cc2185c44a12a0aa57943 ?)

**Steps to reproduce:**
scons p=windows bits=64 tools=no target=release -j 8

**Error Output:**
```Swift
[ 72%] Compiling ==> drivers/gles3/shader_compiler_gles3.cpp
[ 72%] Linking Static Library ==> modules/libmodules.windows.opt.debug.64.a
drivers/wasapi/audio_driver_wasapi.cpp: In member function 'Error AudioDriverWASAPI::init_device(bool)':
drivers/wasapi/audio_driver_wasapi.cpp:154:25: error: 'PKEY_Device_FriendlyName' was not declared in this scope
    hr = props->GetValue(PKEY_Device_FriendlyName, &propvar);
                         ^~~~~~~~~~~~~~~~~~~~~~~~
drivers/wasapi/audio_driver_wasapi.cpp: In member function 'virtual Array AudioDriverWASAPI::get_device_list()':
drivers/wasapi/audio_driver_wasapi.cpp:386:24: error: 'PKEY_Device_FriendlyName' was not declared in this scope
   hr = props->GetValue(PKEY_Device_FriendlyName, &propvar);
                        ^~~~~~~~~~~~~~~~~~~~~~~~
[ 73%] Compiling ==> drivers/gles3/rasterizer_scene_gles3.cpp
[ 73%] Compiling ==> drivers/gles2/rasterizer_canvas_gles2.cpp
scons: *** [drivers/wasapi/audio_driver_wasapi.windows.opt.debug.64.o] Error 1
Ranlib Library         ==> modules/libmodules.windows.opt.debug.64.a
scons: building terminated because of errors.
```

### PR DESCRIPTION
**Pull request by marcelofg55**
Fixes #18046

### CODE REVIEW

File: drivers/wasapi/audio_driver_wasapi.cpp [hunk 1]
```diff
@@ -37,6 +37,25 @@
...
+												e,                 \
+												f,                 \
+												g,                 \
+												h,                 \
+												i,                 \
+												j,                 \
+												k,                 \
+										} },                       \
+		l };
+
```
Comment (vnen) [inline drivers/wasapi/audio_driver_wasapi.cpp]: Why is this formatted like that?

Comment (marcelofg55) [reply drivers/wasapi/audio_driver_wasapi.cpp]: I had to format it like because of clang-format, it was failing otherwise.

Comment (marcelofg55) [reply drivers/wasapi/audio_driver_wasapi.cpp]: Thanks for the help Rémi!

**Comment by hpvb** [comment]
cherry picked into 3.0.3

**Pull request merged**